fixture_magic/handlers.py

```diff
@@ -0,0 +1,33 @@
+from django.conf import settings
+from django.db import models
+from django.utils.module_loading import import_string
+
+FIXTURE_MAGIC_HANDLERS_SETTING_STRING = 'FIXTURE_MAGIC_HANDLERS'
+
+
+def prepare_handlers():
+    handlers = getattr(
+        settings,
+        FIXTURE_MAGIC_HANDLERS_SETTING_STRING,
+        {}
+    )
+    for key in handlers:
+        handlers[key] = import_string(handlers[key])()
+    return handlers
+
+
+class BaseModelHandler:
+    @staticmethod
+    def handle(instance):
```

Comment:
let's add some docstring with info - what actually is handled in this method :smile: